docs/streams/developer-guide/dsl-api.html

```diff
@@ -2849,6 +2851,12 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
                                         <li>When the table is <a class="reference internal" href="#versioned-state-stores"><span class="std std-ref">versioned</span></a>,
                                             the table record to join with is determined by performing a timestamped lookup, i.e., the table record which is joined will be the latest-by-timestamp record with timestamp
                                             less than or equal to the stream record timestamp. If the stream record timestamp is older than the table's history retention, then the record is dropped.</li>
+                                        <li>To use the Grace Period, the table needs to be <a class="reference internal" href="#versioned-state-stores"><span class="std std-ref">versioned</span></a>.
```

Comment:
`[g]race [p]eriod`

not need to capitalize?

Reply:
I could go either way. Lets un cap it for now